apps/frontend/src/app/components/search/user-search.component.html

```diff
@@ -16,7 +16,7 @@
   />
 </div>
 <ng-template #results>
-  <div class="flex min-h-[3.75rem] flex-col justify-center gap-2" [ngClass]="{ 'w-96': useOverlay }">
+  <div class="flex min-h-[3.75rem] flex-col justify-center [&>:not(:first-child)]:mt-2" [ngClass]="{ 'w-96': useOverlay }">
```

Comment:
What's the difference?

Reply:
There's a pagination control or something taking up space in when invisible, I don't remember exactly

Reply:
It doesn't fix it tho. It still adds margin to invisible paginator

Reply:
I'm not sure that's right, I worry that destroying the paginator might lose some important state. But since this is unclear let's not slow the PR down with something unrelated, going to ditch the commit. I don't even think this is a significant issue for a follow-up issue.